PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -120,6 +120,10 @@ class HomeViewModel: NSObject {
     private let recentSavesController: NSFetchedResultsController<SavedItem>
     private let recomendationsController: RichFetchedResultsController<Recommendation>
 
+    private var shouldDisableReader: Bool {
+        featureFlags.isAssigned(flag: .disableReader) || userDefaults.bool(forKey: UserDefaults.Key.toggleOriginalView)
+    }
```

Comment:
Because this property is repeated a few times and it is in fact a combined feature flag itself, what do you think about moving it to `FeatureFlagService`? One way to do it without disrupting the current pattern could be to add an `override` or similarly named property to `CurrentFeatureFlags` which could allow us to add a local override only for the flags that we want (like `disableReader` in this case).

Reply:
I moved this property to `FeatureFlagService` which can be accessed via `featureFlags.shouldDisableReader`. I'm not sure if I follow the local override, but happy to further dive and discussed that.